### DISCUSSION TITLE
Kicked out of console with "Invalid login credentials"?

### DISCUSSION THREAD

**Discussion by Danny-Cisco** [pocketbase/pocketbase#4886] [Q&A] [answered]
My pocketbase is sporadically kicking me out of the console every few hours?
Once it kicks me out, then my password no longer works to get me back in, forcing me to reset my password... until the whole thing happens all over again a few hours later.

Any ideas what is is going on here?

**Comment by ganigeorgiev** [comment]
Without details how you are communicating with PocketBase and what requests you are sending, I don't think anyone will be able to help you.

If you are using an SSR meta framework like SvelteKit, Nuxt, Next.js, etc. then make sure that you have a unique SDK client per each request (see [SSR integration](https://github.com/pocketbase/js-sdk?tab=readme-ov-file#ssr-integration)).

If you need further help, please elaborate a little more on your setup and try to provide some **minimal** code sample to understand what you are executing.

**Reply by Danny-Cisco** [reply]
OK thank you for the lead regarding using a unique client per request. I'm at the stage of making my first crud creates using svelte-kit and pocketbase, so its probably  something like that.

✌️

**Reply by ganigeorgiev** [reply]
I've posted this several times but I don't recommend using a JS SSR framework with PocketBase as most likely you'll mess up the security, especially if you are new to web development.

Alternatively, you may not actually need PocketBase because if you are using SvelteKit with the node adapter you already have a backend, aka. the node server, and if you need just a simple database you can use SQLite directly with Drizzle, Prisma, etc. For auth there are also well documented and maintained libraries like [Lucia](https://github.com/lucia-auth/lucia).

PocketBase is not just a database, it is a backend. The easiest way to use it is by interacting directly with your PocketBase server from the client-side (eg. with a SPA). Some more ramblings on this you can read in https://github.com/pocketbase/pocketbase/discussions/3792#discussioncomment-7620032.

**Comment by Danny-Cisco** [comment, marked as answer]
Thanks for your help. 
My problem was due to Pockethost settings, auto syncing my password to match my pockethost account after waking from hibernation.

But for my next project i will indeed try Lucia and SQlite instead. Thank you
